Jenkinsfile

```
@@ -92,6 +92,9 @@ pipeline {
 
         stage('Acceptance tests') {
             agent { label 'ubuntu && 20.04 && nodejs18' }
+            environment {
+                E2E_TEST_BASE_URL = 'https://platform-acc.publiq.be'
```

Normally we use a trailing slash.
But maybe it will also work like this.

```suggestion
                E2E_TEST_BASE_URL = 'https://platform-acc.publiq.be/'
```